specification/storagemover/resource-manager/readme.python.md

```diff
@@ -3,21 +3,16 @@
 These settings apply only when `--python` is specified on the command line.
 Please also specify `--python-sdks-folder=<path to the root directory of your azure-sdk-for-python clone>`.
 
-``` yaml $(track2)
+``` yaml $(python)
 azure-arm: true
 license-header: MICROSOFT_MIT_NO_VERSION
 package-name: azure-mgmt-storagemover
-no-namespace-folders: true
+namespace: azure.mgmt.storagemover
 package-version: 1.0.0b1
 clear-output-folder: true
 ```
 
-``` yaml $(python-mode) == 'update' && $(track2)
+``` yaml $(python)
 no-namespace-folders: true
```

Comment:
Do we still need this line?

Reply:
Yes. It is related with autorest behavior.